### ISSUE TITLE
Installation using nix fails with fatal error on WSL

### ISSUE THREAD

**Issue by ssenthilanand** [silviogutierrez/reactivated#174]
I have nix setup under WSL2 Debian.

I am at the nix-shell -E "$(curl -L https://reactivated.io/install/)" step.
Python dependencies seem to be installed OK. Then yarn starts adding js packages. It errors out saying

├─ which@2.0.2
├─ word-wrap@1.2.3
├─ yallist@4.0.0
└─ yocto-queue@0.1.0
Done in 14.85s.
fatal: not a git repository (or any of the parent directories): .git

If I go to the newly created project directory and run the app, there are warnings about migrations but I get the homepage at port 8000. Clicking on the polls app errors out:

relation "example_question" does not exist

This seems to be because the setup is not complete yet.

**Comment by silviogutierrez** [comment]
I see the issue: 
https://github.com/silviogutierrez/reactivated/blob/main/packages/create-django-app/scripts/create-django-app.sh#L32

You *don't* have git configured, and it skipped adding a git repo, but then tries to add the files here:
https://github.com/silviogutierrez/reactivated/blob/main/packages/create-django-app/scripts/create-django-app.sh#L35

Easy solution for now: just configure git:

```bash
git config --global user.name "FIRST_NAME LAST_NAME"
git config --global user.email "MY_NAME@example.com"
```

But I will handle this properly in a future release!

**Comment by ssenthilanand** [comment]
The installation works now after setting up git.
It should be mentioned in the webpage for getting started.

**Comment by silviogutierrez** [comment]
#175 handles it gracefully

**Comment by silviogutierrez** [comment]
Fixed by #175
